e2e/e2e.go

```diff
@@ -242,7 +242,38 @@ func provisionNew(ctx context.Context, deps []string, target string) error {
 	return nil
 }
 
+// ensureRunningLocalizerWorks check if a localizer is already running, and if it is
+// ensure it's working properly (responding to pings). If it's not, remove the socket.
+func ensureRunningLocalizerWorks(ctx context.Context) error {
```

Comment:
Don't we already have these two functions written in a public devenv package? I swore it was one of the first things I did when I joined DT

Reply:
It's in localizer itself. I'd go clean up this code to make that shared if we weren't killing it in not too far from now